File: dndtracker/Models/Character.cs
namespace dndtracker.Models;
public class Character : IHasCreator
{


  public string Name { get; set; }
  public int Level { get; set; }
  public int Hp { get; set; }
  public string Img { get; set; }
  public string Classes { get; set; }
  public string SecondClasses { get; set; }
  public string SubClass { get; set; }
  public string Race { get; set; }
  // public AbilityScores AbilityScores { get; set; }
  public string Background { get; set; }
  public string CreatorId { get; set; }
  public Profile Creator { get; set; }
  public int Id { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
  // Populated

}

File: dndtracker.client/src/models/Character.js
export class Character {
  constructor(data) {
    this.id = data.id;
    this.name = data.name;
    this.level = data.level;
    this.img = data.img;
    this.classes = data.classes;
    this.race = data.race;
    this.subclasses = data.subclasses;
    this.abilityScores=data.abilityScores
    this.background=data.background

  }
}
//be able to add higher level spells to be added once you reach that level
//add spell to queue for when you hit that specified level you adds to your spellbook
